packages/block-library/src/file/index.php

```diff
@@ -63,9 +63,13 @@ function register_block_core_file() {
 		)
 	);
 
+	if ( defined( 'IS_GUTENBERG_PLUGIN' ) && IS_GUTENBERG_PLUGIN ) {
+		$module_url = gutenberg_url( '/build/interactivity/file.min.js' );
+	}
+
 	wp_register_script_module(
 		'@wordpress/block-library/file',
-		defined( 'IS_GUTENBERG_PLUGIN' ) && IS_GUTENBERG_PLUGIN ? gutenberg_url( '/build/interactivity/file.min.js' ) : includes_url( 'blocks/file/view.min.js' ),
+		$module_url ?? includes_url( 'blocks/file/view.min.js' ),
```

Comment:
Hmm, is it possible that `$module_url` is undefined here? Would that throw a warning?

Reply:
7 or higher according to the [plugin directory](https://wordpress.org/plugins/gutenberg/). There's [a whole discussion](https://core.trac.wordpress.org/ticket/58874) around whether WP "officially" supports the null coalescing operator or not but I think at this point we should be fine to start using it 😅

Reply:
You can always initialize the `$module_url` before `IS_GUTENBERG_PLUGIN` check to avoid the conditional here.

Reply:
>7 or higher according to the [plugin directory](https://wordpress.org/plugins/gutenberg/). There's [a whole discussion](https://core.trac.wordpress.org/ticket/58874) around whether WP "officially" supports the null coalescing operator or not but I think at this point we should be fine to start using it 😅

The null coalescing operator has not yet been accepted into the WordPress Coding Standards. [There is a Trac ticket open for it](https://core.trac.wordpress.org/ticket/58874). 

Until this ticket or a proposal for it (and other PHP 7.0 syntax) is accepted into Core, please don't use it (yet) for code that will be merged into Core. Why? It will be get removed once it comes to Core.

So to ease the sync process between GB and Core, I'd advise to not use it yet.

Reply:
I thought the null coalescing operator was OK to use in Gutenberg considering its extensive usage: https://github.com/search?q=repo%3AWordPress%2Fgutenberg+path%3A*.php+language%3APHP+%3F%3F&type=code&ref=advsearch.
But I didn't check if it's actually allowed.
I agree with @hellofromtonya, let's not use `??` until it's officially accepted in Core (and appreciate the link to the Trac discussion).
Fixed in 1b61304a422ff9d8bd1f1bbdf30215f3b1ff75ce.

Reply:
> You can always initialize the `$module_url` before `IS_GUTENBERG_PLUGIN` check to avoid the conditional here.

I didn't want to initialize it before the `IS_GUTENBERG_PLUGIN` check for performance-related reasons. But yes, that was an option.